src/PKSim.Core/Services/IndividualTransporterFactory.cs

```diff
@@ -141,22 +149,41 @@ private void addOrganWithLumenParameters(IContainer organ, IndividualTransporter
          var transportDirection = organ.IsInMucosa() ? DefaultMucosaDirectionFor(transporter.TransportType) :
             organ.IsKidney() ? TransportDirectionId.ExcretionKidney : TransportDirectionId.ExcretionLiver;
 
-         addContainerExpression(organ.Container(INTERSTITIAL), transporter, DefaultTissueDirectionFor(transporter.TransportType),
-            FractionParam(FRACTION_EXPRESSED_BASOLATERAL, CoreConstants.Rate.PARAM_F_EXP_BASOLATERAL, editable: false),
+         addContainerExpression(organ.Container(INTERSTITIAL), transporter,
+            DefaultTissueDirectionFor(transporter.TransportType),
+            FractionParam(FRACTION_EXPRESSED_BASOLATERAL, CoreConstants.Rate.PARAM_F_EXP_BASOLATERAL),
             InitialConcentrationParam(CoreConstants.Rate.INITIAL_CONCENTRATION_INTERSTITIAL_TRANSPORTER)
          );
 
          //By default, active transport in mucosa are always apical
          var defaultExpressionApical = organ.IsInMucosa() ? CoreConstants.Rate.ONE_RATE :
             transporter.TransportType == TransportType.Influx ? CoreConstants.Rate.ZERO_RATE : CoreConstants.Rate.ONE_RATE;
 
-         addContainerExpression(organ.Container(INTRACELLULAR), transporter, transportDirection,
+         addContainerExpression(organ.Container(INTRACELLULAR), transporter,
+            transportDirection,
             RelExpParam(REL_EXP),
             FractionParam(FRACTION_EXPRESSED_APICAL, defaultExpressionApical),
             InitialConcentrationParam(CoreConstants.Rate.INITIAL_CONCENTRATION_INTRACELLULAR_TRANSPORTER)
          );
       }
 
+      private void addTissueParameters(IContainer organ, IndividualTransporter transporter)
+      {
+         addTissuePlasmaAndBloodCellsInitialConcentrations(organ, transporter);
+
+         addContainerExpression(organ.Container(INTERSTITIAL), transporter,
+            TransportDirectionId.None,
+            FractionParam(FRACTION_EXPRESSED_BASOLATERAL, CoreConstants.Rate.PARAM_F_EXP_BASOLATERAL, visible: false),
+            InitialConcentrationParam(CoreConstants.Rate.INITIAL_CONCENTRATION_INTERSTITIAL_TRANSPORTER)
+         );
+
+         addContainerExpression(organ.Container(INTRACELLULAR), transporter,
+            DefaultTissueDirectionFor(transporter.TransportType),
+            RelExpParam(REL_EXP),
+            FractionParam(FRACTION_EXPRESSED_APICAL, CoreConstants.Rate.ZERO_RATE)
```

Comment:
@Yuri05 Apical parameter is defined everywhere (before we only add basolateral hidden with value 1)
Now basolateral formula is the same everwhere (1-f_abpical). Default value for apical is 0 and therefor all the same as bfore